app.js

```diff
@@ -5,7 +5,33 @@ var port = process.env.PORT || 4000;
 
 app.get('/', function(req, res) {
     //res.send('<html><head></head><body><h1>Teams:</h1></body></html>');
-    res.send(['acs-cognitive-team', 'acs-fabric-team', 'acs-appdev-team', 'acs-database-team', 'acs-database-db2', 'acs-compute-team', 'acs-security-team', 'acs-network-team', 'acs-storage-team', 'acs-virt-team', 'acsvpc']);
+    res.send(['acs-cognitive-team', 'acs-fabric-team', 'acs-appdev-team', 'acs-database-team', 'acs-database-db2', 'acs-compute-team', 'acs-security-team', 'acs-network-team', 'acs-storage-team', 'acs-virt-team', 'acsvpc', 'acs-paascore']);
 });
 
-app.listen(port, () => console.log(['acs-cognitive-team,acs-fabric-team,acs-appdev-team, acs-database-team, acs-database-db2, acs-compute-team, acs-security-team,acs-network-team, acs-storage-team, acs-virt-team, acsvpc']));
\ No newline at end of file
+let domains = [
+    {
+      "team": 1,
+      "acs-domain": "acs-cognitive",
+      "focus": "service issue and data loss related queries"
+    },
+    {
+      "team": 2,
+      "acs-domain": "acs-fabric",
+      "focus": "service functions"
+    },
+    {
+      "team": 3,
+      "acs-domain": "acs-security",
+      "focus": "service security"
+    }
+  ];
+
+  app.get('/domains', function(req, res) {
+    res.send(domains);
+  });
+
+app.get('/domains/:team', function(req, res) {
+    res.send(req.params);
+});
+
+app.listen(port, () => console.log(['acs-cognitive-team,acs-fabric-team,acs-appdev-team, acs-database-team, acs-database-db2, acs-compute-team, acs-security-team,acs-network-team, acs-storage-team, acs-virt-team, acsvpc, acs-paascore']));
```

Comment:
I think we can delete the callback function which you're providing here as the second argument to `app.listen`